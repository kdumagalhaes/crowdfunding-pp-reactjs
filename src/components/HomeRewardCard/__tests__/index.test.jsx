import { render } from "@testing-library/react";
import { HomeRewardCard } from "..";

describe("HomeRewardCard", () => {
  it("initial component should match snapshot", () => {
    const { asFragment } = render(<HomeRewardCard />);
    expect(asFragment()).toMatchSnapshot();
  });
});
