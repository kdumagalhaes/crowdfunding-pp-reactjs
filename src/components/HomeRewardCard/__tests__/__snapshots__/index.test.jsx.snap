// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeRewardCard initial component should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-gsDKAQ eSZVdg"
    style="opacity: 1;"
  >
    <div
      class="flex align-bottom space-between"
    >
      <h3
        class="title"
      />
      <span
        class="pledge"
      >
        Pledge $ or more
      </span>
    </div>
    <p
      class="paragraph"
    />
    <div
      class="flex space-between align-hor"
    >
      <div
        class="quantity flex align-hor"
      >
        <span
          class="quantity-number"
        />
        <span
          class="quantity-left"
        >
          left
        </span>
      </div>
      <button
        class="sc-bdvvtL kvGkJb"
      >
        Select Reward
      </button>
    </div>
  </div>
</DocumentFragment>
`;

exports[`HomeRewardCard renders like expected 1`] = `
<DocumentFragment>
  <div
    class="sc-gsDKAQ eSZVdg"
    style="opacity: 1;"
  >
    <div
      class="flex align-bottom space-between"
    >
      <h3
        class="title"
      />
      <span
        class="pledge"
      >
        Pledge $ or more
      </span>
    </div>
    <p
      class="paragraph"
    />
    <div
      class="flex space-between align-hor"
    >
      <div
        class="quantity flex align-hor"
      >
        <span
          class="quantity-number"
        />
        <span
          class="quantity-left"
        >
          left
        </span>
      </div>
      <button
        class="sc-bdvvtL kvGkJb"
      >
        Select Reward
      </button>
    </div>
  </div>
</DocumentFragment>
`;
